### PR TITLE
[ASSIST-705]: Implement Flowbite Error Pages

### PR DESCRIPTION
**Pull request by mxm1070**
### Ticket(s) or GitHub Issue

https://engage.canyongbs.com/workgroups/group/3/tasks/task/view/705/

### Technical Description

Flowbite error pages.

### Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] New feature (non-breaking change which adds functionality)
- [x] Content or styling update (Changes which don't affect functionality)

### Screenshots (if appropriate)

Screenshot 2023-11-06 at 5 39 29 PM

Screenshot 2023-11-06 at 5 39 37 PM

### Any deployment steps required?

A deployment step could be a command that needs to be executed or an ENV key that needs to be added, for example.

- [x] No

_______________________________________________

## Before contributing and submitting this PR, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/canyongbs/assistbycanyongbs/blob/main/README.md#contributing).
* [x] Title the PR with the ticket/issue number and a short description of the changes made. Or if no ticket/issue exists, title the PR with a short description of the changes made
* [x] Linked a relevant ticket or issue or describe the issue/feature which this PR resolves/implements.
* [x] Resolved all conflicts, if any.
* [x] Rebased your branch PR on top of the latest upstream `develop` branch.

### CODE REVIEW

**Review by Orrison — COMMENTED** [review]
This looks good to me, but @danharrin if you can give it a final look to see if there is anything Filament specific that we missed here.

**Comment by Orrison** [comment]
Also @mxm1070 there are currently some tests failing.

File: resources/views/components/layout.blade.php
Comment (danharrin) [inline resources/views/components/layout.blade.php:1]: Maybe we can avoid copying this in and consume `<x-filament::panels.layouts.base>`

Comment (mxm1070) [reply resources/views/components/layout.blade.php:1]: I tried this before and I was running into errors like `Unable to locate a class or view for component [filament::panels.layouts.base].`, but also anything related to livewire which is why I copied the layout and stripped out all calls to livewire.

**Review by danharrin — APPROVED** [review]

**Review by Orrison — APPROVED** [review]

**Pull request merged**